climada/hazard/plot.py

```diff
@@ -21,6 +21,9 @@
 
 import logging
 
+import cartopy.crs as ccrs
```

Comment:
I think all the changes to `hazard.plot` are not necessary. Are they maybe leftovers from something else?